src/ngx_http_lua_shdict.c

```diff
@@ -2983,6 +2983,41 @@ ngx_http_lua_ffi_shdict_set_expire(ngx_shm_zone_t *zone, u_char *key,
 
     return NGX_OK;
 }
+
```

Comment:
Style: we need 2 blank lines to separate C function definitions.

Reply:
done